app/model/pairing.py

```diff
@@ -0,0 +1,111 @@
+"""Represent a Pairing between two users."""
+from typing import Dict, Any, TypeVar, Type
+import uuid
+from app.model.base import RocketModel
+
+T = TypeVar('T', bound='Pairing')
+
+
+class Pairing(RocketModel):
+    """Represent a pairing and related fields and methods."""
+
+    def __init__(self,
+                 user1_slack_id: str,
+                 user2_slack_id: str):
+        """
+        Initialize the pairing.
+
+        Pairing ID is a UUID generated by ``uuid.uuid4()``.
```

Comment:
```suggestion
        The generated ``pairing_id`` property is meant to uniquely represent a pairing.
```

something something leaky abstractions :)